api-specs/api-v1.yaml

```
@@ -43,6 +57,16 @@ paths:
                   message:
                     type: string
                     example: Hello YourName
+        400:
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Bad request              
   /api/v1/goodbye:
     get:
       summary: Zwraca wiadomość pożegnalną
```

Please use english for this description
```suggestion
      summary: Return goodbye message
```

---

All messages translated.